iac/compositions/cat-full/service_cat_api.tf

```diff
@@ -0,0 +1,201 @@
+        { name = "CONFIG_EXTERNAL_JAGGAER_BASEURL", value = data.aws_ssm_parameter.parameter["jaggaer-base-url"].value },
+        { name = "CONFIG_EXTERNAL_JAGGAER_CREATEPROJECT_TEMPLATEID", value = data.aws_ssm_parameter.parameter["jaggaer-project-template-id"].value },
+        { name = "CONFIG_EXTERNAL_JAGGAER_CREATERFX_TEMPLATEID", value = data.aws_ssm_parameter.parameter["jaggaer-itt-template-id"].value },
+        { name = "CONFIG_EXTERNAL_JAGGAER_SELFSERVICEID", value = data.aws_ssm_parameter.parameter["jaggaer-self-service-id"].value },
+        { name = "CONFIG_EXTERNAL_NOTIFICATION_USERREGISTRATION_INVALIDDUNSTEMPLATEID", value = data.aws_ssm_parameter.parameter["gov-uk-notify_invalid-duns-template-id"].value },
+        { name = "CONFIG_EXTERNAL_NOTIFICATION_USERREGISTRATION_TARGETEMAIL", value = data.aws_ssm_parameter.parameter["gov-uk-notify_target-email"].value },
+        { name = "CONFIG_EXTERNAL_NOTIFICATION_USERREGISTRATION_TEMPLATEID", value = data.aws_ssm_parameter.parameter["gov-uk-notify_template-id"].value },
+        { name = "CONFIG_EXTERNAL_PROJECTS_SYNC_SCHEDULE", value = data.aws_ssm_parameter.parameter["projects-to-opensearch-sync-schedule"].value },
+        { name = "CONFIG_EXTERNAL_S3_OPPERTUNITIES_SCHEDULE", value = data.aws_ssm_parameter.parameter["oppertunities-s3-export-schedule"].value },
+        { name = "CONFIG_EXTERNAL_S3_OPPERTUNITIES_UI_LINK", value = data.aws_ssm_parameter.parameter["oppertunities-s3-export-ui-link"].value },
+        { name = "CONFIG_FLAGS_DEVMODE", value = var.cat_api_settings.dev_mode },
+        { name = "CONFIG_FLAGS_RESOLVEBUYERUSERSBYSSO", value = var.cat_api_settings.resolve_buyer_users_by_sso },
+        { name = "CONFIG_ROLLBAR_ENVIRONMENT", value = data.aws_ssm_parameter.parameter["rollbar-environment"].value },
+        { name = "ENDPOINT_EXECUTIONTIME_ENABLED", value = var.cat_api_settings.eetime_enabled },
+        { name = "LOGGING_LEVEL_UK_GOV_CROWNCOMMERCIAL_DTS_SCALE_CAT", value = var.cat_api_settings.log_level },
+        { name = "SPRING_PROFILES_ACTIVE", value = "cloud" },
+        { name = "SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_JAGGAER_TOKENURI", value = data.aws_ssm_parameter.parameter["jaggaer-token-url"].value },
+        { name = "SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_JAGGAER_CLIENTID", value = data.aws_ssm_parameter.parameter["jaggaer-client-id"].value },
+        { name = "SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI", value = data.aws_ssm_parameter.parameter["auth-server-jwk-set-uri"].value },
+        { name = "VCAP_SERVICES", value = jsonencode(local.cat_api_vcap_object) },
+      ]
+      essential           = true
+      healthcheck_command = "curl -f http://localhost:8080/health || exit 1"
+      image               = "${module.ecr_repos.repository_urls["cat-api"]}:${var.docker_image_tags.cat_api_http}"
+      // TODO: log groups
+      //      log_group_name      = module.logs.log_group_name
```

Comment:
This is all done for you as part of the [ECS Task Definition module](https://github.com/Crown-Commercial-Service/ccs-migration-alpha-tools/blob/main/resource-groups/ecs-fargate-task-definition/cloudwatch_logs.tf).